CompuCell3D/core/pyinterface/PlayerPythonNew/CMakeLists.txt

```diff
@@ -59,9 +59,13 @@ target_link_libraries(FieldExtractor ${LIBS})
 
 install(TARGETS FieldExtractor
         EXPORT FieldExtractor
-        RUNTIME DESTINATION ${COMPUCELL3D_INSTALL_BIN_DIR}
-        LIBRARY DESTINATION ${COMPUCELL3D_INSTALL_LIB_DIR}
-        ARCHIVE DESTINATION ${COMPUCELL3D_INSTALL_LIB_DIR}
+        RUNTIME DESTINATION bin
```

Comment:
These changes are fine, though I suspect that they create some redundancy in the main module __init__. These nested /bin and /lib paths are currently append on import of cc3d. Should probably verify whether this procedure can be removed due to this proposed change and remove it if so.

Reply:
Additionally, if using the build variables ``COMPUCELL3D_INSTALL_BIN_DIR`` and ``COMPUCELL3D_INSTALL_LIB_DIR`` is eliminated by this PR, then can we remove them altogether in this PR?

Reply:
I would be in favor of eliminating variables we dont use or we can replace by simpler solutions